src/Composer/UsedLicensesParser.php

```diff
@@ -49,4 +49,26 @@ public function getPackagesWithLicense(string $license): array
 
         return $packages;
     }
+
+    /**
+     * @return string[]
+     */
+    public function countPackagesByLicense(): array
+    {
+        $licenses = [];
+        $decodedJson = json_decode($this->retriever->getComposerLicenses(), true);
+        foreach ($decodedJson['dependencies'] as $dependency) {
+            if (isset($dependency['license'][0])) {
+                $licenseName = $dependency['license'][0];
+                if (!isset($licenses[$licenseName])) {
+                    $licenses[$licenseName] = 0;
+                }
+                $licenses[$licenseName]++;
+            }
+        }
+
+        asort($licenses, SORT_NUMERIC);
```

Comment:
Why not arsort? 

https://www.php.net/manual/en/array.sorting.php

Reply:
Because I never bothered learned all sort methods by heart 😂